ai-service/run_tests.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# PYTHONPATHを設定して、テストディレクトリとアプリケーションディレクトリを含める
+export PYTHONPATH=$(pwd):$(pwd)/app
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)